packages/debug/src/common/debug-configuration.ts

```diff
@@ -84,6 +84,12 @@ export namespace DebugConfiguration {
     }
 }
 
+export interface DebugPluginConfiguration {
+    debugMode?: string;
+    pluginLocation?: string;
+    debugPort?: string;
+}
```

Comment:
Any idea why this interface lives in `debug`? 
It seems pretty specific to `plugin-dev` (for hosted-plugins) and not ultimately used by `debug`.